api/src/main/java/org/openmrs/BaseOpenmrsMetadata.java

```diff
@@ -38,7 +39,7 @@ public abstract class BaseOpenmrsMetadata extends BaseOpenmrsObject implements O
 	@Field
 	private String name;
 	
-	@Column(name = "description", length = 255)
```

Comment:
Shouldn't the default length be 255?

Reply:
The default length of varchar is set to 255 by JPA, so there's no need to explicitly define it